app/common/notifications/notify.service.js

```diff
@@ -14,7 +14,8 @@ function Notify(_, $q, $rootScope, $translate, SliderService, ModalService) {
         confirm: confirm,
         confirmModal: confirmModal,
         confirmDelete: confirmDelete,
-        limit: limit
+        limit: limit,
+        importComplete: importComplete
```

Comment:
I don't think this should life in the `notify` service. It isn't a generic notification.. it only relates to importing, so it should live in a service tied to the importer.